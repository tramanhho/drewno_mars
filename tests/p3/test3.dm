main : ( t : int, u : int ) void {
    return;
