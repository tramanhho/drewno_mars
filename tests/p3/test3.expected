syntax error, unexpected end file